ckanext/canada/templates/public/snippets/organization.html

```diff
@@ -25,22 +25,24 @@
     {% endif %}
 
     {% if show_nums %}
-    <div class="row">
-      <div class="col-md-6">
-        <h4 class="text-center">{{ _('Members') }}</h4>
-        <p class="font-xlarge text-center">
-          {{ h.SI_number_span(h.organization_member_count(organization.name)) }}
-        </p>
+      {% set member_count = h.organization_member_count(organization.name) %}
+      <div class="row">
+        {% if member_count >= 0 %}
+          <div class="col-md-6">
+            <h4 class="text-center">{{ _('Members') }}</h4>
+            <p class="font-xlarge text-center">
+              {{ h.SI_number_span(member_count) }}
+            </p>
+          </div>
+          <div class="col-md-6">
+            <h4 class="text-center">{{ _('Datasets') }}</h4>
+            <p class="font-xlarge text-center">
+              {{ h.SI_number_span(organization.package_count) }}
+            </p>
+          </div>
+        {% endif %}
       </div>
```

Comment:
prevent empty row divs?
```suggestion
      {% if member_count >= 0 %}
        <div class="row">
          <div class="col-md-6">
            <h4 class="text-center">{{ _('Members') }}</h4>
            <p class="font-xlarge text-center">
              {{ h.SI_number_span(member_count) }}
            </p>
          </div>
          <div class="col-md-6">
            <h4 class="text-center">{{ _('Datasets') }}</h4>
            <p class="font-xlarge text-center">
              {{ h.SI_number_span(organization.package_count) }}
            </p>
          </div>
        </div>
      {% endif %}
```